### PR TITLE
docs: add disclaimer about NodeJS usage

### PR DESCRIPTION
**Pull request by shivam-deepsource**

### CODE REVIEW

**Comment by tusharsadhwani** [comment]
Is it possible to use node's crypto APIs inplace of web's when running on node, just to make it work everywhere?

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ yarn add @deepsource/shifty
 
 ## Usage
 
+> Shifty is built for the browser and won't work with Node. You can use the built-in [crypto](https://nodejs.org/api/crypto.html#crypto) module instead
```
Comment (yash-deepsource) [inline README.md:25]:
```suggestion
> Shifty is built for the browser and won't work with Node. You can use the built-in [crypto](https://nodejs.org/api/crypto.html#crypto) module instead.
```

**Review by yash-deepsource — APPROVED** [review]

**Comment by yash-deepsource** [comment]
> Is it possible to use node's crypto APIs in place of web's when running on node, just to make it work everywhere?

Considering this option, will get back to you!

**Comment by shivam-deepsource** [comment]
> Is it possible to use node's crypto APIs in place of web's when running on node, just to make it work everywhere?

Currently, we intended for it to be browser only. To be used internally on the frontend app, we can allow node usage if we need it. 

The original rationale was that NodeJS is a more controlled environment, developers already know what APIs are available to them, unlike the browser. One can simply use `crypto.randomBytes(100).toString('base64');` to generate the secret or use something like [nanoid](https://github.com/ai/nanoid)

**Pull request merged**